web_src/js/bootstrap.js

```diff
@@ -6,13 +6,20 @@
 // This file must be imported before any lazy-loading is being attempted.
 __webpack_public_path__ = `${window.config?.assetUrlPrefix ?? '/assets'}/`;
 
-export function showGlobalErrorMessage(msg) {
-  const pageContent = document.querySelector('.page-content');
-  if (!pageContent) return;
+function shouldIgnoreError(err) {
+  const ignorePatterns = [
+    '/assets/js/monaco.', // https://github.com/go-gitea/gitea/issues/30861 , https://github.com/microsoft/monaco-editor/issues/4496
+  ];
+  for (const pattern of ignorePatterns) {
+    if (err.stack.includes(pattern)) return true;
```

Comment:
In line 65 `stack?.` is used. Should we have the null check here too?

Reply:
According to MDN, all browsers support it (we have checked `if (err instanceof Error)` ahead)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack

To keep old behavior, we can add the `?` back.


```suggestion
    if (err.stack?.includes(pattern)) return true;
```